vars_geoportailv3.yaml

```diff
@@ -103,5 +103,6 @@ interpreted:
         - closure_library_path
         cmd: ["node", "-e"]
 
-update_paths:
+update_paths:  # FIXME Why don't we have authtkt and checker in there?
```

Comment:
Sorry I read badly your first question...
for the `authtkt` he isn't present in this file then he can't be update!
and for `checker` I think that's missing...